app/src/androidTest/java/net/skyscanner/backpack/docs/DocsRegistry.kt

```diff
@@ -113,7 +114,7 @@ private fun setupCalendar() {
 }
 
 private fun setupNavBarCollapsed() {
-  Espresso.onView(ViewMatchers.withId(R.id.component_detail_container))
+  Espresso.onView(ViewMatchers.withId(R.id.navBarCoordinator))
```

Comment:
We don't have this scrolling container anymore, switching to CoordinatorLayout located directly in NavBar sample.